.laminas-ci.json

```diff
@@ -0,0 +1,25 @@
+{
+    "extensions": [
+        "bcmath"
+    ],
+    "exclude": [
+        {
+            "name": "PHPUnit on PHP 7.3 with locked dependencies"
```

Comment:
Hmm, these shouldn't be excluded manually: rather, our `composer.json` declares [`php:^8.0.8`](https://github.com/laminas/automatic-releases/blob/f4a0b35bf35db0b8a66adcf9b557d4709e4b533f/composer.json#L7)

The fact that CS runs with something different looks fishy, potentially being an upstream bug :thinking: